lib/OpenQA/Utils.pm

```diff
@@ -899,4 +900,6 @@ sub find_video_files { path(shift)->list_tree->grep(VIDEO_FILE_NAME_REGEX) }
 # workaround https://github.com/mojolicious/mojo/issues/1629
 sub fix_top_level_help { @ARGV = () if ($ARGV[0] // '') =~ qr/^(-h|(--)?help)$/ }
 
+sub is_http_url { shift =~ /http(s)?\:\/\// }
```

Comment:
As we just forward the URL to os-autoinst which calls `git clone` we should not only accept http(s) but anything that git takes. In the commit message you correctly stated just "URL", not that it has to be HTTP(S). Better don't try to roll your own but use something from a common perl library to detect if we have a URL. E.g. use https://docs.mojolicious.org/Mojo/URL

Reply:
I guess we can live with the limitation about the colon

Reply:
Not sure if I understand correctly, so I should change the function like this:
```
my $url = Mojo::URL->new(shift);
return 1 if ($url->scheme);
return 0;
```

Reply:
```suggestion
sub is_url { return !!Mojo::URL->new(shift)->scheme }
```